migrations/V118__Add_ TRB_Request_Forms_Table.sql

```diff
@@ -22,6 +22,92 @@ CREATE TYPE trb_form_status AS ENUM (
     'COMPLETED'
 );
 
+CREATE TYPE trb_technical_reference_architecture_option AS ENUM (
```

Comment:
Editing an existing migration like this is going to cause errors, since this migration's already been run (at least in impl/prod). The added logic from this PR needs to go in a new migration; creating the types is easy enough, altering the `trb_request_forms` table to add the new columns should be easy as well since they're nullable columns.

Reply:
Fixed that